pkg/crypt/crypt.go

```diff
@@ -12,13 +12,17 @@ import (
 
 //EncryptedServiceAccountKey uses github.com/ovotech/mantle to encrypt the
 // key string that's passed in
-func EncryptedServiceAccountKey(key, kmsKey string) (encKey []byte, err error) {
+func EncryptedServiceAccountKey(key, kmsKey string, base64Decode bool) (encKey []byte, err error) {
```

Comment:
It feels slightly bit heavy to add base64Decode to this function, given (a) it's only used in one case, for GitHub, and (b) the encoding isn't really crypt-related.

I wonder if we should leave the signature for EncrypedServiceAccountKey as it was, and make the GitHub Write function do the base-64 encoding to the key before passing it?

Reply:
Agreed. Moved the decoding logic into `pkg/location/github.go`